frappe/core/doctype/doctype/doctype.py

```diff
@@ -1417,8 +1417,9 @@ def validate_data_field_type(docfield):
 					+ "<br>"
 				)
 				df_options_str = "<ul><li>" + "</li><li>".join(_(x) for x in data_field_options) + "</ul>"
+				docfield.options = ""
```

Comment:
There are apps that extend behaviour of fieldtypes by allowing to change options values similar to how Frappe does it (Data::Email, etc)

Reply:
This only applies to `Data` type which you can't change because framework blocks it since quite some time now.

Reply:
Making this non-blocking, I dont see any side-effects. It will just get treated as vanilla `Data` field.